openapi.yaml

```diff
@@ -91,6 +91,18 @@ components:
           type: string
           description: Signed Peer Record to advertise DHT connection information
 
+    Purchase:
+      type: object
+        properties:
+          state:
+            type: string
+            description: Description of the Request's state
+          error:
+            type: string
+            description: If Request failed, then here is presented the error message
+          request:
+            $ref: "#/components/schemas/StorageRequest"
+
```

Comment:
I think this is duplicate as you have renamed `StorageRequestState` into `Purchase` [here](https://github.com/status-im/nim-codex/pull/380/files#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39R180)?

Reply:
Good catch! Duplicate removed.